modules/framework/galasa-parent/dev.galasa.framework.api.openapi/src/main/resources/openapi.yaml

```diff
@@ -2440,6 +2504,55 @@ components:
           description: The fully-qualified property name, including the 'namespace'.
         value:
           type: string
+    GalasaMonitor:
+      type: object
+      properties:
+        apiVersion:
+          type: string
+          description: The API version associated with the GalasaMonitor resource type
+        kind:
+          type: string
+          enum: [GalasaResourceCleanupMonitor]
+        metadata:
+          type: object
+          description: General metadata information about the GalasaMonitor resource
+          properties:
+            name:
+              type: string
+              description: The name of the GalasaMonitor
+            description:
+              type: string
+              description: A description associated with the GalasaMonitor
+        data:
+          type: object
+          description: The data property contains the configuration of the GalasaMonitor
+          properties:
+            isEnabled:
+              type: boolean
+              description: Indicates whether or not the GalasaMonitor is enabled
+            resourceCleanupData:
+              type: object
+              description: Configuration associated with the resource cleanup functions of the GalasaMonitor
+              properties:
+                stream:
+                  type: string
+                  description: |
+                    Optional. The test stream containing resource monitor bundles for the GalasaMonitor to pull in.
+                    By default, the resource management bundles in the dev.galasa.uber.obr bundle will be included.
+                filters:
+                  type: object
+                  description: The filters to apply before loading bundles for the GalasaMonitor to use
+                  properties:
+                    includes:
+                      type: array
+                      description: A comma-separated list of regex patterns representing the bundles that the GalasaMonitor should load
```

Comment:
not regex patterns now. Just in case you missed this one.

Reply:
Addressed in https://github.com/galasa-dev/galasa/pull/237/commits/f451d8dd1e34e1529fc9e157d1170d93284a8066